styles.css

```diff
@@ -0,0 +1,35 @@
+
+#navbar-ul {
+    list-style: none;
+    display: flex;
+    position: fixed;
+    top: 0;
+    height: 50px;
+    background-color: rgb(255, 145, 0);
+    width: 100%;
+    justify-content: space-between;
+}
+#description {
```

Comment:
You should check Linter errors.

Reply:
- [ ] You have done well to use the polite language in the communication, which makes your comment more friendly. However, as the previous reviewer suggested, please ensure to provide the detailed reason why you are requesting for changes. This will give the student a clear understanding of your suggestions and have ability to apply the exact changes required ☺️.